package.json

```diff
@@ -143,6 +143,11 @@
           "type": "boolean",
           "default": false,
           "description": "Enable/disable goto super method code lens."
+        },
+        "metals.enableStripMarginOnTypeFormatting": {
+          "type": "boolean",
+          "default": true,
+          "description": "When enabled, if you press the return key from the first line of a multiline string containing a pipe, it will automatically add .stripMargin."
```

Comment:
Super nitpicky, but instead of using `description` here, you could use `markdownDescription` and then actually put `stripMargin` in backticks which might appear a bit nicer in vscode.

Reply:
ok ! thanks for your review